src/stcal/jump/jump.py

```diff
@@ -537,6 +550,7 @@ def flag_large_events(
     total_snowballs = 0
     nints = gdq.shape[0]
```

Comment:
This could be done as 

`nints, ngroups, nrows, ncols = gdq.shape`

Which would make the next few lines more descriptive in the use of variables.  For example, `gdq.shape[2]` isn't very descriptive.

Reply:
I made the change.